src/client.js

```diff
@@ -93,7 +93,53 @@ export function bootstrap() {
     )
     : new URL(link.getAttribute("href"));
   const client = new Client(initialURL.href);
-  // Register a global listener for history updates.
+
+  // Add click event listener to div#app
+  const appDiv = document.getElementById("app");
+  console.assert(!!appDiv, "bootstrap error: missing div#app element");
+  appDiv.addEventListener("click", (event) => {
+    // Check if the target is an anchor element
+    if (!(event.target instanceof HTMLAnchorElement)) return;
+
+    const anchor = event.target;
```

Comment:
This is too strict, for example the markup could be something like `<a href=""><div class="title">Click me</div></a>` Then the event.target would be the div.title instead of the anchor and this will return.
```suggestion
    const anchor = event.target instanceof HTMLAnchorElement ? event.target : event.target.closest('a');
    if (!anchor) return;
```